soc/arm/nxp_imx/rt/soc_rt10xx.c

```diff
@@ -66,7 +66,12 @@ const clock_enet_pll_config_t ethPllConfig = {
 #else
 	.enableClkOutput25M = false,
 #endif
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(enet), okay)
```

Comment:
Commit message here needs to be to something like- `soc: rt10xx: fix the sequence of Enet2 ref clk enablement`

Reply:
Commit title/message now updated.